wgpu-core/src/device/mod.rs

```diff
@@ -2626,6 +2632,66 @@ impl<B: GfxBackend> Device<B> {
     }
 }
 
+impl Device<crate::backend::Vulkan> {
+    unsafe fn create_raw_vulkan_texture_view(
```

Comment:
This is quite tricky. How do you feel about the approach in https://github.com/gfx-rs/gfx/pull/3761 as an alternative?

Reply:
Recap from discussion we've had on this on Matrix:
The plan right now is to introduce external textures to gfx-rs and use that instead of the raw vulkan views, and see if that's possible with OpenXR.